llvm/include/llvm/Cheerp/PointerAnalyzer.h

```diff
@@ -421,13 +421,14 @@ class PointerAnalysis : public llvm::AnalysisInfoMixin<PointerAnalysis> {
   static llvm::AnalysisKey Key;
 
 public:
+  static llvm::Module* modulePtr;
```

Comment:
This whole thing look weird. What is the purpose?

Reply:
right, removed that commit, thanks,